templates/tabletop-development.ini.erb

```diff
@@ -0,0 +1,6 @@
+# THIS FILE IS GENERATED BY PUPPET
+# DO NOT EDIT DIRECTLY, INSTEAD EDIT THE .erb template
+
+[secret]
```

Comment:
I just want to check real quick - I know there's some shenanigans with plugins and config, is `secret` an enum or anything that'll get _replaced_ by this instead of added to this? I don't expect so, but I want to be careful. @EliAndrewC do you know off the top of your head?